src/pages/MetricDetail.svelte

```diff
@@ -92,6 +91,9 @@
 <style>
   @import "../main.scss";
   @include metadata-table;
+  h2 {
```

Comment:
@Iinh I can't remember if we figured out a way of setting this globally? It would be nice to use a universal value here.

Reply:
You could define this as a mixin in main.scss, then `@include` it anywhere you want to use it.
For example:
```
// main.scss
@mixin table-label {
font-size: 24px; }
```
Then, to use it in this component:
```suggestion
  h2 {
    @include table-label;
  }
```

Reply:
This is definitely better than hardcoding 24px all over the place, but I was thinking more in terms of setting the desired height for `h2` globally (overriding what's defined by protocol), so we don't even have to do this -- I remember we considered using the condensed type scale defined here but it seemed a little small -- https://protocol.mozilla.org/fundamentals/typography.html

I fiddled a bit and think I found something that works-- I think this is better left for a subsequent PR though. For this one, I'll try your approach.